installation.md

```diff
@@ -181,15 +181,17 @@ Schritt 5: Datenbank
 
 ### Schritt 6: Administrator
 
-Nun muss ein Username und ein sicheres Passwort für den Administrator der REDAXO-Installation definiert werden. Sichere Passwörter haben mehr als sechs Zeichen und beinhalten Groß- und Kleinbuchstaben sowie Sonderzeichen. Auch sollte nicht unbedingt `Admin` oder `Administrator` als Benutzername anlegt werden; diese sind zu leicht zu erraten.
+Nun muss ein Username und ein sicheres Passwort für den Administrator der REDAXO-Installation definiert werden. Sichere Passwörter bestehen aus mindestens 8 Zeichen und beinhalten Groß- und Kleinbuchstaben sowie Sonderzeichen. Die Mindestlänge, Anzahl an Groß- und Kleinschreibung sowie die Mindestanzahl an Ziffern können in der `config.yml` Datei im Bereich `password_policy` festgelegt werden. Die maximale Länge von 4096 Zeichen sollte nur mit Bedacht geändert werden. Für tiefergehende Erläuterungen zur Passwortlänge siehe [CVE-2013-5750](https://symfony.com/blog/cve-2013-5750-security-issue-in-fosuserbundle-login-form).
```

Comment:
Evtl hier 12 Zeichen und mehr empfehlen und zufällige Passwörter empfehlen, auch wenn REDAXO weniger Zeichen standardmäßig zulässt.

Reply:
Gibt es eine Quelle mit Hinweisen zur Passwortwahl auf die verlinkt werden könnte? Am besten eine deutschsprachige ... evtl. findet sich beim BSI oder ähnlichem etwas? Zu viel würd ich hier nicht reinnehmen. Wer sich mit solch einem System befasst, sollte schon ein gewisses Grundlagenwissen haben, mMn.

Reply:
https://www.bsi.bund.de/DE/Themen/Verbraucherinnen-und-Verbraucher/Informationen-und-Empfehlungen/Cyber-Sicherheitsempfehlungen/Accountschutz/Sichere-Passwoerter-erstellen/sichere-passwoerter-erstellen_node.html

Reply:
Unabhängig davon, wäre vielleicht eine Idee so etwas zu integrieren: https://github.com/bjeavons/zxcvbn-php
Passt aber vermutlich nicht in den Core

Reply:
Auf das BSI verlinken finde ich tatsächlich hervorragend.

Reply:
Na ja eigentlich streiten sich die Geister wieder was ein sicheres Passwort ist. Aber gut. Ich nehme das jetzt mal auf.  @Koala Vielen Dank